connect-base/2.6/docker-maven-download.sh

```diff
@@ -17,6 +17,7 @@ set -e
 MAVEN_REPO_CENTRAL=${MAVEN_REPO_CENTRAL:-"https://repo1.maven.org/maven2"}
 MAVEN_REPOS_ADDITIONAL=${MAVEN_REPOS_ADDITIONAL:-""}
 MAVEN_REPO_CONFLUENT=${MAVEN_REPO_CONFLUENT:-"https://packages.confluent.io/maven"}
+KAFKA_CONNECT_PLUGINS_DIR=${$KAFKA_CONNECT_PLUGINS_DIR}
```

Comment:
Should here be really twodollar signs?

Reply:
> Should here be really twodollar signs?

No. Fixed!

> What about the case where I want to install to another dir than KAFKA_CONNECT_PLUGINS_DIR but without moving KAFKA_CONNECT_PLUGINS_DIR? I think that is/was the main reason for having it as separate env vars?

IMO, the destination directory should be an argument rather than being inferred from the dependency type. This will make this script simpler and it will be easier to follow where libs are downloaded in the docker images.